samples/notebooks/KubeFlow Pipeline Using TFX OSS Components.ipynb

```diff
@@ -31,7 +31,7 @@
    "outputs": [],
    "source": [
     "# Install Pipeline SDK\n",
-    "!pip3 install https://storage.googleapis.com/ml-pipeline/release/0.1.1/kfp.tar.gz --upgrade\n",
+    "!pip3 install https://storage.googleapis.com/ml-pipeline/release/0.1.2/kfp.tar.gz --upgrade\n",
```

Comment:
I guess this PR has been waiting for too long and the version was actually bumped previously.